### DISCUSSION TITLE
How to get highlighted message pings from only specific users?

### DISCUSSION THREAD

**Discussion by osanity** [Chatterino/chatterino2#2493] [Q&A]
I have message highlight pings enabled for certain streamers I watch, and a few other regular users as well. However, apparently the system seems to only scan through people's usernames, and pings me when someone's username contains the same part of a name of the person I actually want to be pinged by.

For example, let's say I want to be pinged by "Streamer"
So I type in the user highlight settings that I want to be pinged by messages made by "Streamer" and that user only.

Then for some reason, I get pinged by random people that have "Streamer" in their username, like for example "streamerfan" or "123streamer" and I have to blacklist every single one that pops up.

Is there a way to only get pinged by one specific user and that one user only, without getting pinged by random people that contain the same part of their name? Thanks

**Comment by leon-richardt** [comment]
Could you post a screenshot of the settings page with your defined user highlights?
